### PR TITLE
[For 2.6.2] fix(frontend, build scripts): English in logging messages

### LINKED ISSUE #2447: Few french translation issues
## Bug description & symptom

There exist some issue in french translation.

## Expected

1.  "Fichiers déjà téléversés" -> "Fichiers déjà téléversés :" (check ' :' ")

### PR DESCRIPTION
**Pull request by raphj**
Fixes #2447
## Checkpoints

- [ ] Code is clear enough
- [ ] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [ ] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [ ] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

- [ ] Tested by quality team

### CODE REVIEW

**Comment by PhilippeAccorsi** [comment]
@raphj I not understand why this MR is related to issue you add in description because I not seen modification about french translation here.

Can you create another issue and change relation if its not possible to fix french issue.

**Comment by PhilippeAccorsi** [comment]
Soory for my last message. Thats ok.

File: frontend/i18next.scanner/en/translation.json [hunk 1]
```diff
@@ -104,14 +104,14 @@
...
+  "Search…": "Search…",
   "Welcome to Tracim": "Welcome to Tracim",
   "You aren't member of any shared space yet": "You aren't member of any shared space yet",
   "Please refer to an administrator or a trusted user": "Please refer to an administrator or a trusted user",
-  "Have a good day !": "Have a good day !",
+  "Have a good day!": "Have a good day!",
   "Please select a shared space in the left sidebar by clicking on it": "Please select a shared space in the left sidebar by clicking on it",
   "Forgot password": "Forgot password",
-  "Did you forget your password ?": "Did you forget your password ?",
+  "Did you forget your password?": "Did you forget your password ?",
```
Comment (Skylsmoi) [inline frontend/i18next.scanner/en/translation.json]: still a space on the value

File: frontend/i18next.scanner/en/translation.json [hunk 2]
```diff
@@ -226,7 +226,7 @@
   "Create an office document": "Create an office document",
   "Full name (required)": "Full name (required)",
   "Password (required)": "Password (required)",
-  "Thank you, your upload is finished !": "Thank you, your upload is finished !",
+  "Thank you, your upload is finished!": "Thank you, your upload is finished !",
```
Comment (Skylsmoi) [inline frontend/i18next.scanner/en/translation.json]: still a space on the value

**Comment by raphj** [comment]
Strange, leftover spaces were not at the same place in my branch as on your reviews. I did fix them. Thanks for the comments!

**Pull request merged**